docs/source/developer_guide/continuous_integration.rst

```diff
@@ -66,7 +66,7 @@ We recommend that you also set this up locally as it will ensure that you never
 These checks run automatically before you make a new commit.
 To setup, simply navigate to the sktime folder and install our pre-commit configuration:
 
-::
+.. code:: bash
    pre-commit install
```

Comment:
I think you need to add an empty line between `..code` and the command itself. As of now, it looks like this:

![image](https://github.com/sktime/sktime/assets/39331844/32771951-4f19-4295-abfc-72c179ae4e3c)

Ref. https://sktime--5377.org.readthedocs.build/en/5377/developer_guide/continuous_integration.html#code-quality-checks

Reply:
Hi @yarnabrina i made changes and removed the file